.env.example

```
@@ -0,0 +1,24 @@
+# Database
+DB_NAME=career-scout
+DB_USER=postgres
+DB_PASSWORD=  # See secrets/db_password.txt
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)